libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -849,7 +849,7 @@ void AP_TECS::_update_throttle_without_airspeed(int16_t throttle_nudge)
     _pitch_demand_lpf.apply(_pitch_dem, _DT);
     const float pitch_demand_hpf = _pitch_dem - _pitch_demand_lpf.get();
     _pitch_measured_lpf.apply(_ahrs.pitch, _DT);
-    const float pitch_corrected_lpf = _pitch_measured_lpf.get() - radians(0.01f * (float)aparm.pitch_trim_cd);
+    const float pitch_corrected_lpf = _pitch_measured_lpf.get() - radians(aparm.pitch_trim);
```

Comment:
Because the old method was broken this is a change in behavior that will need testing.